benchcheck_test.go

```diff
@@ -2,17 +2,23 @@ package benchcheck_test
 
 import (
 	"os"
+	"path/filepath"
+	"strings"
 	"testing"
 
 	"github.com/madlambda/benchcheck"
 )
 
 func TestGetModule(t *testing.T) {
+	// Not extremely happy with the way the tests are coupled to changes
```

Comment:
I don't quite understand this comment. Do you mean you don't want to make tests here as coupled as tests in jtoh?
If so, I think you should remove this comment and add a GitHub comment for that. But if you do believe it's important to keep in the source, then rephrase it with the reasoning but not mentioning other projects because at the time other people are going to read this maybe tests on `jtoh` had being rewritten completely.
Also "Not extremely happy" is too personal :)

Reply:
its not important, will remove